.github/workflows/issues-workflow.yaml

```diff
@@ -13,7 +13,14 @@ jobs:
     name: Add issue to project
     runs-on: ubuntu-latest
     steps:
+      - uses: svrooij/secret-gate-action@v1
+        id: secret
+        with:
+          inputsToCheck: "ADD_ISSUES_TOKEN"
+        env:
+          ADD_ISSUES_TOKEN: ${{ secrets.ADD_ISSUES_TOKEN }}
       - uses: actions/add-to-project@v0.5.0
+        if: steps.secret.outputs.inputsChecked == 'true'
```

Comment:
did you try the following?? 

```
if: secrets.ADD_ISSUES_TOKEN != ''
```
Maybe it works (I have not tried myself)

Reply:
You might want to check this https://github.com/Kuadrant/kuadrant-operator/pull/385

Implementing the same for the kuadrant opeator